src/app.ts

```diff
@@ -3,16 +3,30 @@ import cors from "cors";
 import fileRouter from "@routes/files.route";
 import studyplanRouter from "@routes/studyplans.route";
 import resourceRouter from "@routes/resources.route";
+import cacheEndpoint from "@middlewares/rediscache"; 
+import { getAllStudyPlans } from '@controllers/studyplans.controller';
```

Comment:
following the removal of line 14 (`app.get('/api/resources/study_plans', getAllStudyPlans);`), this import can be removed too

Reply:
still not removed na @NayHtetKyaw